test/e2e/registrymirror.go

```diff
@@ -13,8 +13,8 @@ func runRegistryMirrorConfigFlow(test *framework.ClusterE2ETest) {
 	test.ExtractDownloadedArtifacts()
 	test.DownloadImages()
 	test.ImportImages()
-	test.CreateCluster()
-	test.DeleteCluster()
+	test.CreateCluster(framework.WithBundlesOverride("./eks-anywhere-downloads/bundle-release.yaml"))
```

Comment:
nit: can you use this var instead https://github.com/aws/eks-anywhere/blob/main/test/e2e/airgap.go#L20